.cargo/config.toml

```diff
@@ -1,7 +1,7 @@
 [alias]
-ci-build-all = "build --all-targets --all-features"
-ci-build-inx = "build --all-targets --no-default-features --features inx,stardust"
-ci-build-api = "build --all-targets --no-default-features --features api-core,api-history,analytics,stardust"
+ci-check-all = "check --all-targets --all-features"
+ci-check-inx = "check --all-targets --no-default-features --features inx,stardust"
+ci-check-api = "check --all-targets --no-default-features --features api-core,api-history,analytics,stardust"
```

Comment:
If we do this, we also need to remove the `--release` flags from the tests (which we probably should do anyways).

Reply:
Actually no. If we'ld do that then the tests would run significantly slower.

Reply:
Why do you think the above change should also mean removing `--release` from `ci-test`? If you think it would reuse the `build` it didn't because previously it was a `debug` build anyway, not a `release` build, so `ci-test` compiled an extra release build.

Reply:
You're right—I got confused there. The thing is right now the `--release` build is probably more expansive than what we gain from running the tests a little bit faster, which is why we should probably remove the `--release` from the tests for the time being.